config/resources/defaults.yaml

```diff
@@ -14,21 +14,24 @@
 # limitations under the License.
 #
 
-Port: 8443
-WebPort: 8444
-WebAddress: "0.0.0.0"
-ManagerPort: 1213
-SummaryMonitoringPort: 9931
-DetailedMonitoringPort: 9930
-MonitoringPortLower: 9930
-MonitoringPortHigher: 9999
-Mount: ""
-NamespacePrefix: ""
 Debug: false
+WebAddress: "0.0.0.0"
+Server:
+  WebPort: 8444
```

Comment:
Let's change:
- `WebAddress` -> `Server.Address`
- `WebPort` -> `Server.Port`
The `Web` prefix was added to distinguish this from the Xrootd server's port number; this is currently achieved by putting it in the `Server` section.